Dockerfile

```diff
@@ -0,0 +1,19 @@
+# phase-build
+FROM node:12-alpine as builder
```

Comment:
as builder?

Reply:
That command is used to distinguish phase. Every `FROM` will be builded in different images.
The `FROM xxx as builder` and `COPY --from builder` is used to copy files in different images.

Reply:
👍